python-package/lightgbm/plotting.py

```diff
@@ -2,6 +2,7 @@
 """Plotting library."""
 from copy import deepcopy
 from io import BytesIO
+from typing import Any, List, Dict, Tuple, Optional, Union
```

Comment:
```suggestion
from typing import Any, Dict, List, Optional, Tuple, Union
```

can you please alphabetize this list? I think that makes it a bit easier to keep track of what is being imported.